doc/async-file-upload.md

```diff
@@ -1,6 +1,6 @@
 # Asynchronous file upload
 
-In case of [POST Workspace Layers](rest.md#post-workspace-layers) and [PATCH Workspace Layer](rest.md#patch-workspace-layer), it is possible to upload data files asynchronously, which is suitable for large files. Let's demonstrate how it can be implemented on client side.
+In case of [POST Workspace Layers](rest.md#post-workspace-layers) and [PATCH Workspace Layer](https://github.com/LayerManager/layman/blob/v2.1.0/doc/rest.md#patch-workspace-layer), it is possible to upload data files asynchronously, which is suitable for large files. Let's demonstrate how it can be implemented on client side.
```

Comment:
I think, the `[PATCH Workspace Layer]` should still lead to current documentation.